src/domain/action/templates.ts

```diff
@@ -0,0 +1,48 @@
+const ACTION_TEMPLATES = [
```

Comment:
est-ce que ça mériterait d'être mis en db ?

Reply:
Ça l'est pas depuis le début, j'avoue qu'en codant ça vendredi fin d'aprem j'ai eu la flemme, mais je vois pas trop l'intérêt pour l'instant.

Reply:
je sens quand même que c'est qqch qui vit donc bon en in memory c'est pas ouf

On peut se dire que au prochain change à la demande du métier, ca passe en db (à materialiser par un commentaire dans le code)